src/framework/audio/isequencer.h

```diff
@@ -34,10 +34,10 @@ class ISequencer : MODULE_EXPORT_INTERFACE
 public:
     virtual ~ISequencer() = default;
 
-    using midi_track_t = std::shared_ptr<IMIDIPlayer>;
-    using audio_track_t = std::shared_ptr<IAudioPlayer>;
-    using track_t = std::shared_ptr<IPlayer>;
-    using track_id = unsigned int;
+    using MidiTrack = std::shared_ptr<IMIDIPlayer>;
+    using AudioTrack = std::shared_ptr<IAudioPlayer>;
```

Comment:
I think, better to rename those aliases. They should have the same name as interfaces:

MidiTrack -> IMidiPlayerPtr
AudioTrack -> IAudioPlayerPtr
etc

Reply:
Not sure.
These types are not aliases to `std :: shared_ptr <IMIDIPlayer>`, they are types belonging to the mixer, that is, we may decide in the future to do so
```
using MidiTrack = std::shared_ptr<MyMidiTrack>;
```
for the mixer, it will still be `MidiTrack`, but it will no longer be `IMidiPlayerPtr`